controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -245,28 +277,28 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(ctx context.Context
 	}
 
 	enactmentConditions.NotifyMatching()
-	if !instance.Spec.Parallel {
-		enactmentCount, err := r.enactmentsCountsByPolicy(instance)
-		if err != nil {
-			log.Error(err, "Error getting enactment counts")
-			return ctrl.Result{}, nil
-		}
-		if enactmentCount.Failed() > 0 {
-			err = fmt.Errorf("policy has failing enactments, aborting")
-			log.Error(err, "")
-			enactmentConditions.NotifyAborted(err)
-			return ctrl.Result{}, nil
-		}
-		err = r.claimNodeRunningUpdate(instance)
-		if err != nil {
-			if apierrors.IsConflict(err) {
-				return ctrl.Result{RequeueAfter: nodeRunningUpdateRetryTime}, err
-			} else {
-				return ctrl.Result{}, err
-			}
+
+	enactmentCount, err := r.enactmentsCountsByPolicy(instance)
+	if err != nil {
+		log.Error(err, "Error getting enactment counts")
+		return ctrl.Result{}, nil
```

Comment:
We don't return the error ? It will call Reconcile again.

Reply:
AFAIK, returning error triggers reconcile again
https://cluster-api.sigs.k8s.io/developer/providers/implementers-guide/controllers_and_reconciliation.html#reconciliation

Reply:
But we really don't want Reconcile to be called again ?

Reply:
Hmm, indeed we should reconcile.

Reply:
Fixed